debugger/pythonExtension/gdb_sgx_plugin.py

```diff
@@ -268,8 +268,11 @@ def stop(self):
         # Get untrusted stack frame pointer and corresponding TCS.
         frame_pointer = int(gdb.parse_and_eval("$rdi"))
         tcs_addr = int(gdb.parse_and_eval("$rsi"))
+        enclave_base_addr = int(gdb.parse_and_eval("$rdx"))
+        gs_base = read_int_from_memory(tcs_addr + TCS_GSBASE_OFFSET, POINTER_SIZE)
+
         # Get callsite of the TCS.
-        td_addr = tcs_addr + TD_OFFSET_FROM_TCS
+        td_addr = enclave_base_addr + gs_base
```

Comment:
Very clean change. Do you happen to know if there is some history as to why we used a different method originally?

Reply:
@johnkord I am not sure about the history.